solvers/H3LAPD/EquationSystems/DriftReducedSystem.hpp

```diff
@@ -61,6 +61,8 @@ class DriftReducedSystem
   Array<OneD, Array<OneD, NekDouble>> Evec;
   /// Storage for ExB drift velocity
   Array<OneD, Array<OneD, NekDouble>> ExB_vel;
+  /// Mesh dimension, stored as a member var for convenience
+  int n_dims;
```

Comment:
Minor: "n_" seems to alias "number" and "density".

Reply:
I don't disagree, but after merging from main this has moved to EqnSysBase, I'll file an issue to remind myself to change it later.